johnsonandjohnson/Bodiless-JS
# Deprecrate Search from vital demo sites

## Changes
Remove Search Dependency from vital site

## Test Instructions
Run all sitesb below in dev mode and confirm search is removed in header and no valid /search pages
Build & serve all static sites and confirm search is removed from header & no /search pages is 404 
*Note some of these can be tested directly from the vercel deploy and gave you the link

List of sites dependent on vital packages:

- sites/__vital__: https://bodiless-js-starter-git-fork-hvanyo-fix-c90922-jnj-demo-account.vercel.app/?vercelToolbarCode=_JmuqI4gNyvlsoU
- sites/__vital_next__
- sites/vital_demo: https://bodiless-js-gatsby-git-fork-hvanyo-fix-6df192-jnj-demo-account.vercel.app/
- sites/vital-demo-next: https://bodiless-js-next-git-fork-hvanyo-fix-re-c97e5a-jnj-demo-account.vercel.app/?vercelToolbarCode=cBXG44PTgBvnUiO
- sites/vital-examples: https://bodiless-js-examples-git-fork-hvanyo-fi-438be1-jnj-demo-account.vercel.app/?vercelToolbarCode=MFVJWFaQCwBoNWl


## Related Issues
